lib/ReferenceFinder/Tests/Unit/LocationTest.php

```diff
@@ -3,18 +3,18 @@
 namespace Phpactor\ReferenceFinder\Tests\Unit;
 
 use PHPUnit\Framework\TestCase;
-use Phpactor\ReferenceFinder\DefinitionLocation;
 use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\Location;
 use Phpactor\TextDocument\TextDocumentUri;
 
-class DefinitionLocationTest extends TestCase
+class LocationTest extends TestCase
```

Comment:
this test no longer maps to a class, we can either simply remove it or move the test cases to the test for `TextDocument\Location`

Reply:
Removing since there already is a test for the location in the TextDocument namespace.